client/hostinfo_unix.cpp

```diff
@@ -2203,35 +2205,97 @@ long xss_idle() {
 }
 #endif // HAVE_XSS
 
+/**
```

Comment:
We do not use Doxygen on the project, please convert this comments block to the regular comments

Reply:
Done.